wdtk-wikibaseapi/src/main/java/org/wikidata/wdtk/wikibaseapi/WikibaseDataEditor.java

```diff
@@ -36,6 +36,28 @@
  */
 public class WikibaseDataEditor {
 
+	/**
+	 * Field to store current csrfToken.
+	 */
+	static String csrfToken;
```

Comment:
Same issue here - no reason to have this static. The CSRF token is already stored in the `WbEditingAction` instance, why do you need to duplicate it here?

Reply:
ok i got it.